core/templates/dev/head/pages/exploration_player/LearnerLocalNav.js

```diff
@@ -92,7 +92,8 @@ oppia.controller('LearnerLocalNav', [
         });
       });
     };
-
+    $scope.canEdit = GLOBALS.canEdit;
```

Comment:
Leave a blank line after line 94.

Reply:
@shubha1593, Is it in the proper place? (like in between function) I guess defining them initially would be better (Line 30).